docs/getting-started/github-apps/migrating-tools-to-github-apps.rst

```diff
@@ -1,153 +1,99 @@
 
-Once the GitHub app is installed on the correct repo, the next step is to create a ``/.dockstore.yml`` file. We'll cover a very straightforward example
+The next step is to create a .dockstore.yml file. We'll cover a very straightforward example
 first, but depending on how you configured the tool during registration and whether your GitHub repository houses multiple tools published on Dockstore,
-there will be additional steps to writing your ``/.dockstore.yml`` file.
+there will be additional steps to writing your .dockstore.yml file.
 
 Let's say we have the following CWL tool registered on Dockstore that came from this `repository <https://github.com/kathy-t/dockstore-tool-helloworld>`__ and you would like to convert the master branch.
 
 .. figure:: /assets/images/docs/single-tool-to-migrate.png
    :alt: Tool to Migrate
 
-As noted in our other documentation, create a ``/.dockstore.yml`` file in the root directory of the branch you want to migrate (in this example, it's the master branch) in your repository. The file should look like the following:
+As noted in our other documentation, create a .dockstore.yml file in the root directory of the branch you want to migrate (in this example, it's the master branch) in your repository. The file should look like the following:
 
-.. code:: yaml
-
-   version: 1.2
-   tools:
-      - subclass: CWL
-        primaryDescriptorPath: /Dockstore.cwl
-        testParameterFiles:
-            - /test.json
+.. include:: /assets/templates/tools/example-1-noname.yml
+  :code:
 
 The information above was filled out using the following:
 
 - ``subclass`` is based on the descriptor language your tool is described in. In this case, the tool is described in CWL.
-- ``primaryDescriptorPath`` is from ``CWL Path`` or ``WDL Path``, depending on the ``subclass``. 
-- ``testParameterFiles`` is from ``CWL Test Parameter File Path`` or ``WDL Test Parameter File Path``, depending on the ``subclass``.
-
-Existing Dockstore tools can be described in two languages, CWL and WDL, but GitHub App tools can only be described in one language. If your existing tool is described in two languages, you can either pick one language
-or register two GitHub App tools, one for each language. This can be accomplished using one ``.dockstore.yml``. It may look like the following:
+- ``primaryDescriptorPath`` is the path to the CWL descriptor file.
+- ``testParameterFiles`` is the path to any test parameter files.
 
-.. code:: yaml
+The ``name`` Field
+~~~~~~~~~~~~~~~~~~
 
-   version: 1.2
-   tools:
-      - subclass: CWL
-        primaryDescriptorPath: /Dockstore.cwl
-        testParameterFiles:
-            - /test.json
-      - subclass: WDL
-        primaryDescriptorPath: /Dockstore.wdl
-        testParameterFiles:
-            - /test.json
-        name: wdl-tool
+If you have more than one tool in the same repository, the ``name`` field must be filled out for at least one of the tools to ensure that the tool paths are unique.
```

Comment:
I know this is hard to word, and I have no concrete suggestion, but it's not at least one of the tools, e.g., if there are 3 tools, you need the name on at least 2 of the tools.

Reply:
Yes, it does imply that, but I believe it is okay to imply that because the alternative is using a lot of space to explain that it isn't *strictly* required on provided n-1 of them have names... the way I see it, what a user gains from that explanation is just one line of text. Unless users have something to actually gain from having one workflow go unnamed while allowing others to be named, I don't think this is a distinction that is worth making in this section. It's an extra detail that can easily confuse people and is more likely to cause errors (if you have ten workflows, and decide you do not want to name one of them, and then add an 11th, you might not name the 11th either which results in an error -- but if you are in the habit of naming every tool this would not happen).

Reply:
Except for the Broad :), I think most authors will have one tool or one workflow per .dockstore.yml, so telling the name is  required seems wrong.

Also I'm not sure which text you're referring to, the existing text, or your proposed revision.

Reply:
Sorry, I'm confused what the suggestion is. I'm not telling them it is required. The sentence starts with "If you have more than one tool in the same repository," and we have examples that do not have the name field. 

What's in this PR is a sentence that is somewhat implying (incorrectly) that *every* tool in a multi-tool repo needs a name. The truth is only n-1 tools need a name, but I do not think there is a quick way to explain that in this particular introductory documentation, and attempt to do so would open the door for confusion and errors.

Maybe it's best we make this distinction in the template with the maximum amount of text? That way it is still being said, but it is a detail that is not immediately clear to novice users who do need to use it.

(All of what I'm saying here assumes that there is nothing to gain by having one tool not include name in a multi-tool dockstore.yml, of course. As far as I'm aware the only gain is not having to type about a dozen characters. If that assumption is incorrect then let me know so I can communicate the pros of it to users.)